README.rst

```diff
@@ -71,6 +71,9 @@ Some functions require some extra packages:
 
    $ pip install biotite
 
+After install, biotite's Chemical Component Dictionary (CCD) can be updated via
```

Comment:
The README is rather a short summary of the most important points. Could you rather adapt the respective documentation page (https://github.com/biotite-dev/biotite/blob/main/doc/install.rst)?

Reply:
ok! removed from the main readme and made minor edits to the install.rst. i'll leave to you to make other edits there as I'm a bit unclear on the source installation procedure